restli-common/src/main/java/com/linkedin/restli/common/ContentType.java

```diff
@@ -107,10 +111,12 @@
         new SymbolTableBasedContentTypeProvider(PROTOBUF,
```

Comment:
Update this codec also.

Reply:
updated, left it initially since it is deprecated.